api/document/tests/tree_test.py

```diff
@@ -128,6 +128,37 @@ def test_parent():
         == 'root_0__sect_2__par_a'
 
 
+def test_ancestors():
+    root = tree.DocCursor.new_tree('root', '0')
+    root.add_child('sect')
```

Comment:
I realize this is only a test, and that `'sect'` might as well be `'foo'`, but insofar as tests function a bit like documentation, do you think this should be `'sec'` just to stay consistent with the names we use in our actual schema?  I know I personally have become confused about whether our real schema's paragraphs are called `'par'` or `'para'` due to inconsistencies like this.

Um, speaking of which, should this test use `'para'` instead of `'par'` too? 😅

Reply:
That's a good point; I'll search-and-replace.